instagram/src/components/AppContainer/PostContainer/PostComponents/CommentContainer/CommentContainer.js

```diff
@@ -0,0 +1,98 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+import IconsLikes from "./IconsLikes";
+import Comments from "./Comments/Comments";
+import AddComment from "./AddComment";
+
+const CommentSection = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+`;
+
+class CommentContainer extends Component {
+  state = {
+    comments: [],
+    comment: "",
+    likes: 0,
+    liked: false,
+    showComment: false
+  };
+
+  componentDidMount() {
```

Comment:
You don't need to use CDM to spread comments from prop to the comment in state. you can set the comments in state to props.comments. Same with likes.